src/main/kotlin/com/atlassian/performance/tools/jiraactions/w3c/harvesters/JsResources.kt

```diff
@@ -63,12 +66,31 @@ private fun parsePerformanceEntry(
 }
 
 private fun parseServerTimings(
-    jsServerTimings: Any
+    jsServerTimings: Any,
+    jsExecutor: JavascriptExecutor
 ): List<PerformanceServerTiming> {
     if (jsServerTimings !is List<*>) {
         throw Exception("Unexpected non-list JavaScript value: $jsServerTimings")
     }
-    return jsServerTimings.map { parseServerTiming(it) }
+    val result = jsServerTimings.map { parseServerTiming(it) }.toMutableList()
+    val responseHeaders = getResponseHeaders(jsExecutor)
+    val nodeId = responseHeaders["x-anodeid"]
+    val requestId = responseHeaders["x-arequestid"]
+    addAttribute(result, "nodeId", nodeId)
+    addAttribute(result, "requestId", requestId)
```

Comment:
Not sure if I like the fact that we are artificially putting product specific data into W3C model and that it happens in `w3c` package, not in some higher level wrapper. This creates a bit of a chaos of where things belong.

The main problem with this that I see is that if the Jira specific data will become API we won't be able to change our mind about this model. The only way will be fork or API break.

Reply:
our API is `BackendTimeSlot`, javascript field is a technical detail